include/core/Window.hpp

```diff
@@ -36,14 +36,14 @@ class Window : public RenderTarget {
 		void close() { m_isOpen = false; }
 		bool isOpen() const { return m_isOpen; }
 
-		SDL_Window *window() const { return m_window.get(); }
+		sf::RenderWindow *window() const { return m_window.get(); }
 
 	private:
-		using SDL_WindowPtr = std::unique_ptr<SDL_Window, decltype(&SDL_DestroyWindow)>;
-		using SDL_GLContextPtr = std::unique_ptr<void, decltype(&SDL_GL_DeleteContext)>;
+		using SFML_WindowPtr = std::unique_ptr<sf::RenderWindow>;
+//		using SFML_GLContextPtr = std::unique_ptr<sf::Context>;  //For multithreading?
 
-		SDL_WindowPtr m_window{nullptr, SDL_DestroyWindow};
-		SDL_GLContextPtr m_context{nullptr, SDL_GL_DeleteContext};
+		SFML_WindowPtr m_window{nullptr};
+//		SFML_GLContextPtr m_context{nullptr};
```

Comment:
Here, `SFML_WindowPtr` is completely useless, the only use of `SDL_WindowPtr` was to handle the pointer to the SDL_Window, but in SFML you don't get a pointer, you can use directly `sf::RenderWindow`

Reply:
You can also remove the lines you commented, I don't plan to use `sf::Context` yet

Reply:
Done, using `sf::RenderWindow` directly now.